packages/trader/src/Modules/CFD/Containers/cfd-password-modal.jsx

```diff
@@ -598,7 +598,15 @@ const CFDPasswordModal = ({
 
     const handleForgotPassword = () => {
         closeModal();
-        const redirect_to = platform === CFD_PLATFORMS.MT5 ? 1 : 2;
+        let redirect_to = platform === CFD_PLATFORMS.MT5 ? 1 : 2;
+
+        // if account type is real convert redirect_to from 1 or 2 to 10 or 20
+        // and if account type is demo convert redirect_to from 1 or 2 to 11 or 21
+        if (account_type.category === 'real') {
+            redirect_to = Number(`${redirect_to}0`);
+        } else if (account_type.category === 'demo') {
+            redirect_to = Number(`${redirect_to}1`);
+        }
```

Comment:
Why are we having this changes?

Reply:
@matin-binary  Because they want us to keep the previous demo/real tab in the redirection link sent by email so I had to add new redirection types to the `redirect_to` value. when it is `1` it means it is `/mt5`. but when it is `10` it means it is `/mt5#real`
and when it is `11` it means it is `/mt5#demo`. the same logic is applied to trade too.